ci_scripts/check_documentation.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+log_file="${TMPDIR}/jazzy_status.log"
+
+if ! command -v jazzy > /dev/null; then
+  if [[ "${CI}" != "true" ]]; then
+    echo "ERROR: Please install jazzy:"
+    echo "https://github.com/realm/jazzy"
+    exit 1
+  fi
+
+  echo "Installing jazzy..."
+
+  gem install jazzy || die "Executing \`gem install jazzy\` failed"
```

Comment:
Sorry, I meant that `die` isn't a built in function, it's something we write as a helper to shorten implementations. Same reasoning behind `info` which attaches information about what script is generating the log line. Does that clear it up?